### PR TITLE
enable foreman and katello modules on el8

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: roles/foreman_repositories/tasks/redhat.yml [hunk 1]
```diff
@@ -18,13 +18,40 @@
...
+    warn: False
+    creates: /etc/dnf/modules.d/katello.module
+  when:
+    - foreman_repositories_katello_version is defined
+    - foreman_repositories_katello_version == "nightly" or foreman_repositories_katello_version is version('4.4', '>=')
+    - ansible_distribution_major_version == '8'
+
 - when:
     - foreman_repositories_katello_version is defined
     - ansible_distribution_major_version == '8'
```
Comment (ekohl) [inline roles/foreman_repositories/tasks/redhat.yml]: Shouldn't this be scoped to `< 4.4`? At least, `pki-core` (and possibly powertools`) shouldn't be needed anymore, right?

Comment (evgeni) [reply roles/foreman_repositories/tasks/redhat.yml]: pki-core true, powertools are still needed (thats a repo, not a module)

but I wanted to keep the diff small.

**Comment by evgeni** [comment]
```
Error: Problems in request:", "missing groups or modules: katello:el8
```

:thinking:

**Comment by evgeni** [comment]
fascinating, the 3.2 repos really don't have module metadata on the CDN, but whyyyy
(they do on koji)

**Comment by evgeni** [comment]
https://github.com/theforeman/foreman-infra/blob/f847559002fc18bffc759b190a712aa934a60af0/puppet/modules/web/files/deploy-yumrepo.sh#L50

:crying_cat_face:

File: roles/foreman_repositories/tasks/redhat.yml (hunk 1 above)
Comment (evgeni) [reply roles/foreman_repositories/tasks/redhat.yml]: implemented now

**Review by ehelms — APPROVED** [review]

**Pull request merged**